server/models/Room.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Room",
+  "plural": "Rooms",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "options": {
+    "validateUpsert": true,
+    "mongodb": {
+      "collection": "wye-rooms"
+    }
+  },
+  "properties": {
+    "id": {
+      "type": "string",
+      "required": true
+    },
+    "Country": {
+      "type": "string"
+    },
+    "Region": {
+      "type": "string"
+    },
+    "City": {
+      "type": "string"
+    },
+    "Organization": {
```

Comment:
Let's use "Organisation" to be british ;-) 

May be a more complex structure at some point.